pkgs/tools/misc/trillian/gomod2nix.toml

```diff
@@ -0,0 +1,167 @@
+schema = 3
+subPackages = ["cmd/trillian_log_server", "cmd/trillian_log_signer", "cmd/createtree", "cmd/deletetree", "cmd/updatetree"]
+goPackagePath = "github.com/google/trillian"
```

Comment:
- Why is this needed? 
- Why can't we infer this from the `src`? 
- Why does it need to be in this file?

Reply:
There is no `src` attribute explicitly being set. 
The `goPackagePath` is there so the derivation knows which sources to select for the `src` attribute.

You could separate the `src` if you wanted to and do the whole manual hash update dance, nothing stops you from doing that but that will leave you without an update script.